pkg/controller/status/controller_test.go

```
@@ -254,55 +254,70 @@ func Test_updatingConditionsFromDegradedToDisabled(t *testing.T) {
 	assert.Equal(t, disabledCondition, getConditionByType(updatedCO.Status.Conditions, OperatorDisabled))
 }
 
-func Test_shouldSetProgressingCondition(t *testing.T) {
+func Test_checkVersionChanges(t *testing.T) {
 	tests := []struct {
-		name                    string
-		newVersion              string
-		clusterOperatorVersions []configv1.OperandVersion
-		expectedShouldUpdate    bool
-		expectError             bool
+		name                         string
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)